### ISSUE TITLE
`rig install --without-p3m` doesn't work as expected

### ISSUE THREAD

**Issue by lewinfox** [r-lib/rig#193]
From reading [the docs](https://github.com/r-lib/rig#-faq-) I expect `rig install --without-p3m` to leave the default package repos alone. However, this option doesn't appear to do anything.

This is a similar question to #87, but I'm asking about the behaviour of the flag rather than of rig in general.

Example Dockerfile to reproduce what I'm seeing:

``` Dockerfile
FROM ubuntu:latest

RUN apt-get update && apt-get -y install curl
RUN curl -Ls https://github.com/r-lib/rig/releases/download/latest/rig-linux-$(arch)-latest.tar.gz | tar xz -C /usr/local

RUN rig install --without-p3m release
RUN rig default release

CMD rig run -e "getOption('repos')"
```

``` shell
docker build -t rig-demo . && docker run --rm rig-demo
```

Expected output:

```
                         CRAN 
"https://cloud.r-project.org" 
```

Actual output:

```
                                                          P3M 
"https://packagemanager.posit.co/cran/__linux__/jammy/latest" 
                                                         CRAN 
                                "https://cloud.r-project.org" 
```

Removing the `--without-p3m` flag gives identical output from `getOption('repos')`.

Am I misunderstanding the purpose of the flag or is this a bug?

This is an awesome tool, thanks very much!

**Comment by gaborcsardi** [comment]
Yeah, that seems like a bug, unfortunately. Thanks for the great reproducible example!

**Comment by gaborcsardi** [comment]
Closed by 382bed1cccd6693981101fe4614bd9fabf509ebd
